dataland-frontend/src/components/resources/frameworkDataSearch/FrameworkDataSearchFilters.vue

```diff
@@ -0,0 +1,197 @@
+<template>
+  <div class="flex">
+    <div class="flex flex-column">
+      <span class="d-section-heading mb-2" v-if="showHeading">Filter by company</span>
+      <div>
+        <FrameworkDataSearchDropdownFilter
+          v-model="selectedSectorsInt"
+          :available-items="availableSectors"
+          filter-name="Sector"
+          filter-id="sector-filter"
+          filter-placeholder="Search sectors"
+        />
+        <FrameworkDataSearchDropdownFilter
+          v-model="selectedCountriesInt"
+          :available-items="availableCountries"
+          filter-name="Country"
+          filter-id="country-filter"
+          filter-placeholder="Search countries"
+          class="ml-3"
+        />
+      </div>
+    </div>
+    <div class="flex flex-column ml-3">
+      <span class="d-section-heading mb-2" v-if="showHeading">Filter by data</span>
+      <div class="flex flex-row align-items-center">
+        <div class="d-separator-left" />
+        <FrameworkDataSearchDropdownFilter
+          v-model="selectedFrameworksInt"
+          :available-items="availableFrameworks"
+          filter-name="Framework"
+          filter-id="framework-filter"
+          filter-placeholder="Search frameworks"
+          class="ml-3"
+        />
+        <div class="d-separator-left ml-3" />
+        <span class="ml-3 cursor-pointer text-primary font-semibold d-letters" @click="resetFilters">RESET</span>
+      </div>
+    </div>
+  </div>
+</template>
+
+<style lang="scss" scoped>
+.d-section-heading {
+  text-align: left;
+  font-size: 0.75rem;
+  color: #5a4f36;
+}
+
+.d-separator-left {
+  height: 2rem;
+  border-left: 1px solid #5a4f36;
+}
+</style>
+
+<script lang="ts">
+import { defineComponent, inject } from "vue";
+import Keycloak from "keycloak-js";
+import { ApiClientProvider } from "@/services/ApiClients";
+import { getCountryNameFromCountryCode } from "@/utils/CountryCodes";
+import FrameworkDataSearchDropdownFilter, {
+  SelectableItem,
+} from "@/components/resources/frameworkDataSearch/FrameworkDataSearchDropdownFilter.vue";
+import { DataTypeEnum } from "@clients/backend";
+import { humanizeString } from "@/utils/StringHumanizer";
+import { useRoute } from "vue-router";
+
+interface CountryCodeSelectableItem extends SelectableItem {
+  countryCode: string;
+}
+
+interface FrameworkSelectableItem extends SelectableItem {
+  frameworkDataType: DataTypeEnum;
+}
+
+export default defineComponent({
+  name: "FrameworkDataSearchFilters",
+  components: { FrameworkDataSearchDropdownFilter },
+  emits: ["update:selectedCountryCodes", "update:selectedFrameworks", "update:selectedSectors"],
```

Comment:
Wofür gibt es diese emits? Auf sie wird nirgendwo in der parent-Component reagiert.

Reply:
Nehme es zurück.  Sie sind notwendig für das v-model in der parent-Component